### PR TITLE
Compile under Xcode 9 / Swift 3.2

### PR DESCRIPTION
**Pull request by Dimillian**
This is some very lite change I've done to make CryptoSwift compile under the new Xcode and Swift 3.2 compiler.

### CODE REVIEW

**Comment by krzyzanowskim** [comment]
Thank you! awesome. Don't get it wrong, I won't merge the changes. I already started to work on it by myself and I hope not all changes are needed though

**Comment by Dimillian** [comment]
Of course, this just for me, need my project to compile ;)

File: Sources/CryptoSwift/AES.swift [hunk 1]
```diff
@@ -197,12 +197,27 @@ fileprivate extension AES {
...
+        let r8 = UInt8(b2 & 0xFF)
+        let r9 = UInt8((b2 >> 8) & 0xFF)
+        let r10 = UInt8((b2 >> 16) & 0xFF)
+        let r11 = UInt8((b2 >> 24) & 0xFF)
+        let r12 = UInt8(b3 & 0xFF)
+        let r13 = UInt8((b3 >> 8) & 0xFF)
+        let r14 = UInt8((b3 >> 16) & 0xFF)
+        let r15 = UInt8((b3 >> 24) & 0xFF)
         return [
-            UInt8(b0 & 0xFF),UInt8((b0 >> 8) & 0xFF),UInt8((b0 >> 16) & 0xFF),UInt8((b0 >> 24) & 0xFF),
```
Comment (krzyzanowskim) [inline Sources/CryptoSwift/AES.swift]: hm... this shouldn't be needed. why is it so? because "it's too complicated to blabla" ?

Comment (gonghao) [inline Sources/CryptoSwift/AES.swift]: Yes. I think it's for the "too complicated to blabla".

Comment (Dimillian) [inline Sources/CryptoSwift/AES.swift]: Yeah, it may be a compiler bug in B1, would be good to report it to Apple.

**Comment by krzyzanowskim** [comment]
Swift 3.2 and Xcode 9 compatibility is available from the `develop` branch

**Comment by SpacyRicochet** [comment]
No chance of making this an actual release, is there?

We actually use CryptoSwift as part of a private pod, but podspecs don't like you pointing them towards `:git` URLs and branches.

**Comment by krzyzanowskim** [comment]
hm.... I could do beta release maybe. Hold my 🍺

**Comment by SpacyRicochet** [comment]
Did you get a chance to make a beta release on the `develop` branch? Would be awesome.

Meanwhile, we've forked it to manage our own Swift 3.2 compatibility, but `pod lib lint` is complaining about frameworks or tons of build errors. Very confusing.

**Comment by krzyzanowskim** [comment]
yes. Can't push to CocoaPods though due to backend error 500 :-( reported to CocoaPods

**Comment by SpacyRicochet** [comment]
Ah, that's too bad. I wanted to let you know that we got it working through a fork and a private pod repo now. So we're good now.

The release is probably still great for other people, so I can only encourage you to continue trying :)

And thanks for looking into this for us! ❤️

**Comment by krzyzanowskim** [comment]
🚀  CryptoSwift (0.7.0-alpha.1) successfully published

**Pull request closed without merging**